### DISCUSSION TITLE
My attempt at a planner for the Supernote A6X2

### DISCUSSION THREAD

**Discussion by jasonkul** [kudrykv/latex-yearly-planner#103] [Show and tell]
I should say right up front that my only experience with Latex is reading a howto post by mvgambino and consulting Dr. Google. Definitely a trial and error experience. I wanted to create a slimmed down planner for the A6X2:

- Removed title, quarterly, weekly and daily reflections pages
- Reduced number of note indexes and numbered notes
- Increased line spacing and hyperlink and header sizes
- Slimmed down daily page to just calendar, todo and notes
- Made monthly calendar size a little larger
- Adjusted configs to make everything more usable on a smaller screen 

[sn_a6x2.mos.default.dailycal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13814668/sn_a6x2.mos.default.dailycal.pdf)

Here are the files I wrecked to create it:

[sn_a6x2.mos.default.dailycal.zip](https://github.com/kudrykv/latex-yearly-planner/files/13814670/sn_a6x2.mos.default.dailycal.zip)

**Comment by jasonkul** [comment]
Dotted version
[sn_a6x2.mos.default.dotted.dailycal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13836063/sn_a6x2.mos.default.dotted.dailycal.pdf)

**Reply by radish0e** [reply]
This is what I need with a schedule. I love this template but it's so small on A6X

**Reply by kudrykv** [reply]
Check this one out
[index.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13927327/index.pdf)

**Reply by radish0e** [reply]
Oh wow, thank you!!

**Reply by id628** [reply]
> Dotted version [sn_a6x2.mos.default.dotted.dailycal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13836063/sn_a6x2.mos.default.dotted.dailycal.pdf)

Love this one. I don't need a daily schedule, my Google Calendar can handle that.

Will you update for 2025?

**Reply by id628** [reply]
Love this design, trying to build it for 2025, but not having great luck:

`
E:\x temp\latexplanner\latex-yearly-planner>go run cmd/plannergen/plannergen.go --config "cfg/sn_a6x2.base.yaml,cfg/sn_a6x2.template_months_on_side.yaml,cfg/sn_a6x2.mos.default.yaml,cfg/sn_a6x2.mos.default.dailycal.yaml" "sn_a6x2.mos.default.dailycal"
execute notes_indexed on mos_09_a6x2_notes.tpl:
  execute template:
    template: no template "mos_09_a6x2_notes.tpl" associated with template ""
exit status 1
`

**Reply by tekmuscle** [reply]
I was able to create one for 2025: https://github.com/kudrykv/latex-yearly-planner/discussions/103#discussioncomment-11680259

**Reply by singularity2022** [reply]
@id628 I solved it by simply renaming the mos_09_notes in the tpl folder to "mos_09_a6x2_notes.tpl" and changing the header to the a6x2 version. The index page is still sized for the regular one, but it's good enough, expecially considering that it's only the index pages anyway. Hope this helps.

**Reply by petertung** [reply]
> Check this one out [index.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13927327/index.pdf)

Thank you so much for this!  Would there be 2026 and 2027 versions available?

**Comment by tekmuscle** [comment]
> Check this one out [index.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13927327/index.pdf)

This is great, just started using it with my A6X2. Are the files out there to modify (lined, am/pm, etc)? Would be happy to take a crack at that.

**Reply by kudrykv** [reply]
Uhh, there's nothing readily available yet. I'm on my fourth (?) iteration of refactor 😬 Hopefully, that one would be final.
You can check out the `rubify` branch, has the code to generate the planner — but I'm reslicing it in the `rubify2`.

**Reply by tekmuscle** [reply]
Right on! Appreciate your work. So far this has been smooth on the A6X2.

**Reply by ngoc-saiko** [reply]
@kudrykv I'm using your generated file in Boox tab mini c and it's great. Could you guide me how to do the generation? I want to make it for the next years.

**Comment by singularity2022** [comment]
Is there a 2025 version? I tried to make it using your zip file but I get an error - 
 _template: no template "mos_09_a6x2_notes.tpl" associated with template ""_

Not sure if I'm doing something wrong - I have no idea how to code, but I'm just following the instructions on [this thread](https://github.com/kudrykv/latex-yearly-planner/discussions/94).

**Reply by canrilo** [reply]
I got the same error :( I removed the mention from the tpls file, however my PDFs are being exported with many errors. I'd like to make a breadcrumb template for my A6X2 but so far it has been unsuccessful (even trying with the files provided for MOS).

If anyone has the cfg files for any of these two would be awesome :)

**Reply by canrilo** [reply]
Update: spent so much time learning how to modify the basic mos that when I tried to create a bc I didn't like it at all. Kept the mos version.

**Comment by tekmuscle** [comment]
I did some cranking on the rubify2 branch today and got it working for the A62X, based on the index.pdf @kudrykv posted above. It's for 2025 and I put it in 12 hour format with days of the week starting on Monday. Anyone who follows this discussion, feel free to give it a go.

[sn_a6x2.dotted.12.mon.2025.pdf](https://github.com/user-attachments/files/18263955/sn_a6x2.dotted.12.mon.2025.pdf)

**Reply by kohlsudduth** [reply]
Most importantly, thank you for giving your time to this. I truly appreciate it. It's great and usuable as is. That said, if you're looking for feedback to refine it, the dots are very very faint. Practically speaking, they're not visible on the device. However, they are visible when viewed on my computer screen. Would you consider trying a version with the dots larger and darker?

**Reply by tekmuscle** [reply]
Making the dots bigger might be past my abilities as it's not a parameter in the yaml. Might be something for @kudrykv to consider and could be a issue to log.

**Reply by VspiderXV** [reply]
Any chance you can share the configuration files please?, trying to do some tweaks of my own but can't quite seem to get what to modify and they would be awesome as reference, thanks.

**Reply by tekmuscle** [reply]
Here's the config file I used. 
[119x158-rightie.tekmuscle.yaml.zip](https://github.com/user-attachments/files/18277988/119x158-rightie.tekmuscle.yaml.zip)

**Reply by id628** [reply]
How do you have it not show the daily schedule itself? I live by Google Calendar, I want to use this just for notes, todos, meetings.

**Reply by singularity2022** [reply]
Is that what you have in mind? [sn_a6x2.mos.default.dailycal.pdf](https://github.com/user-attachments/files/18283877/sn_a6x2.mos.default.dailycal.pdf)

**Reply by VspiderXV** [reply]
How were you able to remove the quarterly pages?, every time i set the section to "false" it throws a bunch of errors about missing links

Heres the section from the config file:

    - quarterly:
        enabled: false
        parameters:
          months_width: 4.3cm
          gap_width: 5mm
          reverse_columns: false

This is an example of the error:

`error: label <quarter-2025-1> does not exist in the document`

**Reply by singularity2022** [reply]
I just used @jasonkul config files on the top of this thread. It errors out because it's missing one of the templates, so I just renamed the corresponding regular template to the one I needed and changed the header.

**Reply by id628** [reply]
> Is that what you have in mind? [sn_a6x2.mos.default.dailycal.pdf](https://github.com/user-attachments/files/18283877/sn_a6x2.mos.default.dailycal.pdf)

Perfect!

**Reply by petertung** [reply]
> Is that what you have in mind? [sn_a6x2.mos.default.dailycal.pdf](https://github.com/user-attachments/files/18283877/sn_a6x2.mos.default.dailycal.pdf)

This is great!  Would you be able to make 2026 and 2027 versions of this calendar?

**Reply by tekmuscle** [reply]
Here's the 2026 version of what I setup for 2025. 
[sn_a6x2.dotted.12.mon.2026.pdf](https://github.com/user-attachments/files/24443637/sn_a6x2.dotted.12.mon.2026.pdf)

**Comment by canrilo** [comment]
After several hours diggings first in this post and then in the depths of the code, making countless experiments and understanding what makes what, I was way too satisfied with my result and forgot to come back to the beginning. 

Initially, I was kinda displeased with broken hyperlinks and some extra details in the versions posted in this thread, which motivated my research. Here's my final product for the Nomad A6X2. Dotted, with daily cals, weeks, extra notes, reflects, tears and love :) 
[sn_a6x2.mos.default.dailycal.pdf](https://github.com/user-attachments/files/20683578/sn_a6x2.mos.default.dailycal.pdf)
